benchmark/Streamly/Benchmark/Data/Fold.hs

```diff
@@ -57,174 +55,151 @@ moduleName = "Data.Fold"
+              value
+              "pipe-mapM"
+              (S.fold
+                   (IFL.transform
+                        (Pipe.mapM (\x -> return $ x + 1))
+                        FL.drain))
         ]
     ]
 
 o_1_space_serial_composition :: Int -> [Benchmark]
 o_1_space_serial_composition value =
-    [ bgroup "serially"
-        [ bgroup "composition" -- Applicative
-            [ benchIOSink
-                  value
-                  "all,any"
-                  (S.fold
-                       ((,) <$> FL.all (<= (value + 1)) <*>
-                        FL.any (> (value + 1))))
-            , benchIOSink
-                  value
-                  "sum,length"
-                  (S.fold ((,) <$> FL.sum <*> FL.length))
-            , benchIOSink
-                  value
-                  "tee sum length"
-                  (S.fold (FL.tee FL.sum FL.length))
-            , benchIOSink
-                  value
-                  "distribute sum length"
-                  (S.fold (FL.distribute [FL.sum, FL.length]))
-            , benchIOSink
-                  value
-                  "partition sum length"
-                  (S.fold (IFL.lmap
-                               (\a -> if odd a then Left a else Right a)
-                               (FL.partition FL.sum FL.length)))
-            , benchIOSink
-                  value
-                  "unzip sum length"
-                  (S.fold (IFL.lmap (\a -> (a, a)) (FL.unzip FL.sum FL.length)))
-            ]
+    [ bgroup "composition" -- Applicative
+        [ benchIOSink
+              value
+              "all,any"
+              (S.fold
+                   ((,) <$> FL.all (<= (value + 1)) <*>
+                    FL.any (> (value + 1))))
+        , benchIOSink
+              value
+              "sum,length"
+              (S.fold ((,) <$> FL.sum <*> FL.length))
+        , benchIOSink
+              value
+              "tee sum length"
+              (S.fold (FL.tee FL.sum FL.length))
+        , benchIOSink
+              value
+              "distribute sum length"
+              (S.fold (FL.distribute [FL.sum, FL.length]))
+        , benchIOSink
+              value
+              "partition sum length"
+              (S.fold (IFL.lmap
+                           (\a -> if odd a then Left a else Right a)
+                           (FL.partition FL.sum FL.length)))
+        , benchIOSink
+              value
+              "unzip sum length"
+              (S.fold (IFL.lmap (\a -> (a, a)) (FL.unzip FL.sum FL.length)))
         ]
     ]
 
-o_n_heap_serial :: Int -> [Benchmark]
-o_n_heap_serial value =
-    [ bgroup "serially"
-          [ bgroup "elimination"
-          -- Left folds for building a structure are inherently non-streaming
-          -- as the structure cannot be lazily consumed until fully built.
-                [ benchIOSink value "toStream" (S.fold IP.toStream)
-                , benchIOSink value "toStreamRev" (S.fold IP.toStreamRev)
```

Comment:
Only the above two benchmarks were to be moved, not all of them. Because toStream and toStreamRev are defined in the Streamly.Internal.Data.Stream.IsStream, whose benchmarks are in Prelude.Serial benchmark module.

Reply:
Oh okay, I'll fix it.